src/styles/blocks/card.scss

```diff
@@ -0,0 +1,113 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  background: $color-background;
+  border: 1px solid #e5e5e5;
+  border-radius: 5px;
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    padding-bottom: 16px;
+  }
+
+  &__price-text {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-text-code;
+  }
+
+  &__price-value {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $color-text-name;
+  }
+
+  &__reviews {
+    display: flex;
+    align-items: center;
+  }
+
+  &__reviews-text {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $color-text-name;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    padding-bottom: 24px;
+  }
```

Comment:
![2023-02-15_15h32_35](https://user-images.githubusercontent.com/61904995/219041401-044ea1bb-67e1-4323-a62f-431a9365ee17.png)
It should be margin